n2y/main.py

```diff
@@ -21,6 +21,8 @@ def main():
     parser.add_argument("--image-path", help="Specify path where to save images")
     parser.add_argument("--image-web-path", help="Web path for images")
     parser.add_argument("--plugins", help="Plugin file")
+    parser.add_argument("--target", '-t', default='./',
```

Comment:
You may want to look at some other command line tools to see what other people use to indicate the output directory's location. When possible, it's nice to make the command line arguments line up with what other tools do. You may have done this already.

Reply:
Although modern versions of windows _should_ be fine with the `/` instead of `\` for a directory separator, I'd still suggest changing the default from `./` to just `.` since that seems cleaner and devs may think that `./` is invalid.